PasswordValidator/Controllers/PasswordValidationController.cs

```diff
@@ -4,14 +4,13 @@ namespace PasswordValidator.Controllers
 {
     public class PasswordValidationController : Controller
     {
-        readonly SimpleValidatorFactory _SimpleValidatorFactory = new SimpleValidatorFactory();
-        readonly AdvancedValidatorFactory _AdvancedValidatorFactory = new AdvancedValidatorFactory();
+        readonly ValidatorFactory _validatorFactory = new ValidatorFactory();
```

Comment:
Need to look at Dependency Injection - https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection?view=aspnetcore-6.0